cg/services/orders/store_order_services/store_case_order.py

```diff
@@ -34,263 +38,182 @@ def __init__(
         self.status_db = status_db
         self.lims = lims_service
 
-    def store_order(self, order: OrderIn) -> dict:
+    def store_order(self, order: OrderWithCases) -> dict:
         """Submit a batch of samples for sequencing and analysis."""
         return self._process_case_samples(order=order)
```

Comment:
I think this is an unnecessary wrapping. The `store_order` of the other services handle everything directly in their body